api/publish.go

```diff
@@ -16,8 +16,8 @@ import (
 type signingParams struct {
 	// Don't sign published repository
 	Skip bool `                json:"Skip"           example:"false"`
-	// GPG key ID to use when signing the release, if not specified default key is used
-	GpgKey string `            json:"GpgKey"         example:"A0546A43624A8331"`
+	// GPG key ID(s) to use when signing the release, CSV if multiple keys, if not specified default configured key(s) are used
+	GpgKey string `            json:"GpgKey"         example:"KEY_ID_a,KEY_ID_b"`
```

Comment:
do we allow a space after the comma ? I would add it in the example if yes, looks nicer